doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,197 @@ Table 7-2 shows security capabilities
+| sec.img.009 | Images **must** only be accessible to authorized actors |  | |
+| sec.img.010 | Image Registries **must** only be accessible to authorized actors |  | |
+| sec.img.011 | Image Registries **must** only be accessible over secure networks |  | |
+| sec.img.012 | Image registries **must** be clear of vulnerable and stale (out of date) versions |  | |
+
+<a name="7.11.6"></a>
+### 7.11.6. Security LCM
+
+| Ref | Requirement | Unit | Definition |
+|---|----|---|----|
+| sec.lcm.001 | The Platform **must** support Secure Provisioning, Maintaining availability, Deprovisioning (secure Clean-Up) of workload resources |  | Secure clean-up: tear-down, defending against virus or other attacks, or observing of cryptographic or user service data |
+| sec.lcm.002 | Operational **must** use management protocols limiting security risk such as SNMPv3, SSH v2, ICMP, NTP, syslog and TLS |  | |
+| sec.lcm.003 | The Cloud Operator **must** implement change management for NFVI, VRM and other components of the cloud |  | Platform change control on hardware |
+| sec.lcm.004 | The Cloud Operator **should** support automated templated approved changes |  | Templated approved changes for automation where available |
+| sec.lcm.005 | Platform **must** provide logs and these logs must be regularly scanned |  | |
+| sec.lcm.006 | The Platform **must** verify the integrity of all Resource management requests | Yes/No | |
+| sec.lcm.007 | The Platform **must** be able to update newly instantiated, suspended, hibernated, migrated and restarted images with current time and time zone information |  | |
+| sec.lcm.008 | The Platform **must** be able to update newly instantiated, suspended, hibernated, migrated and restarted images with relevant DNS information. |  | |
+| sec.lcm.009 | The Platform **must** be able to update newly instantiated, suspended, hibernated, migrated and restarted images with relevant IP addressing, including routing tables, route health information and whitelists/blacklists. |  | |
+| sec.lcm.010 | The Platform **must** be able to update newly instantiated, suspended, hibernated, migrated and restarted images with relevant geolocation information |  | |
+| sec.lcm.011 | The Platform **must** log all changes to geolocation along with the mechanisms and sources of location information (i.e. GPS, IP block, and timing). |  | |
+| sec.lcm.012 | The Platform **must** implement Security life cycle management processes including proactively update and patch of all deployed software. | | |
+
+<a name="7.11.7"></a>
+### 7.11.7. Monitoring and Security Audit
+
+The Platform is assumed to provide configurable alerting and notification capability and the operator is assumed to have automated systems, policies and procedures to act on alerts and notifications in a timely fashion. In the following the monitoring and logging capabilities can trigger alerts and notifications for appropriate action.
```

Comment:
This section may represent significant scope creep
- full log monitoring, analysis, alerting
- incoming traffic analysis capabilities, DDOS protection
- detection of 'illegal' access
- automated response ("the platform must [...] take corrective security hardening measures")

This section should be re-visited to see how to discern log management & integrity functions that are required of the NFVI, from the requirements on a log management system, from the capabilities of Intrusion Detection System, from the capabilities of an Intrusion Prevention System, DDOS mitigation, and so on.

Reply:
@kedmison The LCM chapter has a section of Logging and monitoring (LMA). This section is providing requirements for some of what LMA is providing.